pkg/component/etcd/bootstrap.go

```diff
@@ -375,8 +359,6 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	resources["crd.yaml"] = []byte(CRD)
-	resources["crdEtcdCopyBackupsTask.yaml"] = []byte(etcdCopyBackupsTaskCRD)
```

Comment:
This will make GRM try to delete the CRDs, right?
I guess we have to keep them with the ignore annotation for at least one release.